src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -23,7 +23,7 @@ public class DurableTaskOptions
         /// <summary>
         /// Settings used for Durable HTTP functionality.
         /// </summary>
-        public HttpOptions HttpSettings { get; set; }
+        public HttpOptions HttpSettings { get; set; } = new HttpOptions();
```

Comment:
I ran into a NullReferenceException in my CallHttpAsync manual test. HttpOptions was never given a value. This fixes that bug that I found.